src/app/issues/[id]/page.tsx

```diff
@@ -0,0 +1,33 @@
+import AddViewPointBar from "@/components/Conversation/AddViewPointBar";
+import IssueCard from "@/components/Conversation/IssueCard";
+import {
+    mockIssue,
+    mockViewPointList,
+    mockEmptyIssue,
+} from "@/mock/conversationMock";
+import ViewPointList from "@/components/Conversation/ViewPointList";
+
+type IssueViewProps = {
+    params: {
+        id: string;
+    };
+};
+
+export default function IssueView({ params }: IssueViewProps) {
+    const { id } = params;
+    console.log(id);
```

Comment:
That is because if I keep the params there and didn’t use it, next won’t let me build the project.
And I want to keep the function structure with params for future data fetching feature.